### DISCUSSION TITLE
VPS server requirements for a with 20K visitors and 40K visits

### DISCUSSION THREAD

**Discussion by Piskatje13** [plausible/analytics#4083] [Self-Hosted Support]
Hello,

What are VPS server (minimum) requirements for a website with 20K visitors and 40K visits? https://ibb.co/yBSZGkZ
Will be 1 vCPU + 2gb enough?

**Comment by ruslandoga** [comment]
👋 @Piskatje13 

https://github.com/plausible/analytics/discussions/3852#discussioncomment-8779245 might be useful.

**Comment by Piskatje13** [comment]
Thanks for your answer. I have only 200 visitors/300 visits per day. I have installed Plausible on a free tier of Google Cloud (1vCPU, 1gb). Plausible keeps going down.  Tried to install on Ubuntu 22.04, Debian, AWS free tier. The result is the same. Plausible keeps going down but the server does not..

**Reply by ruslandoga** [reply]
👋 @Piskatje13 

Could you post your `docker compose logs`? It might help us understand why it is going down.

**Comment by Piskatje13** [comment]
I have moved my Plausible to another VPS with 4 vCPU, 6 GB RAM (Contabo). It works now. 

Many Thanks!!! 

PS: Any tips for security? I installed Plausible on a Debian VPS with a Cloudflare tunnel. Is it enough?
